docs/configuring-metabase/caching.md

```diff
@@ -7,50 +7,125 @@ redirect_from:
 
 ### Caching per database
 
-You can override your default caching options for each database connection, caching the results for more or less time than the default time-to-live (TTL) duration set by your site-wide settings. Setting caching per question is especially useful when data relevant to the question has a different natural cadence than your site-wide caching rule.
+This setting tells Metabase how long to keep the cached results from a specific database.
+
+1. Make sure [caching is enabled](#enabling-global-caching).
+2. Go to **Admin settings** > **Databases** and select your database.
+3. Open **Advanced options** and find the **Default result cache duration**.
+4. Click **Custom** and enter a cache duration in hours.
 
-Go to **Admin settings** > **Databases** and select your database connection. Under **Advanced settings**, set the **Default result cache duration**, which determines how long to keep question results for that database. By default, Metabase will use the value you supply on the [cache settings page](#caching-settings), but if this database has other factors that influence the freshness of data, it could make sense to set a custom duration. You can also choose custom durations on individual questions or dashboards to help improve performance.
+This setting is useful for databases that take longer to query, or databases that are kept up to date on a special cadence. 
+
+This setting will override your [global TTL settings](#cache-time-to-live-ttl-multiplier).
 
 ### Caching per question
 
-You can override your default caching options for questions, caching the results for more or less time than the default time-to-live (TTL) duration set by your site-wide caching settings. Setting caching per question is especially useful when data relevant to the question has a different natural cadence than your site-wide caching rule, such as when the question queries data that doesn't change often.
\ No newline at end of file
+This setting tells Metabase how long to keep the cached results for a specific question. You'll only find these cache settings on questions that exceed the [minimum query duration](#minimum-query-duration).
+
+1. Make sure [caching is enabled](#enabling-global-caching).
+2. Go to your question.
+3. Click on the **info** icon.
+4. Click **Cache configuration**.
+5. Enter a cache duration in hours.
+6. Click **Save changes**.
+
+You can use this setting to update questions on the same cadence as your data. For example, if your data gets updated daily, you can set the **Cache configuration** to 24 hours.
+
+If set, your question cache duration will override:
+
+- [global cache duration](#cache-time-to-live-ttl-multiplier)
+- [database cache duration](#caching-per-database)
+
+### Caching per dashboard
+
+This setting tells Metabase how long to keep the cached results for each of the questions on a dashboard.
```

Comment:
```suggestion
You can tell Metabase how long to keep the cached results for each of the questions on a dashboard.
```

Editing these slightly as these aren't really single-location settings, like in the admin panel; "these settings" isn't quite right; and the rephrasing makes it seem like you can communicate with Metabase.